### DISCUSSION TITLE
Custom meta<T> without value

### DISCUSSION THREAD

**Discussion by SenseOnline** [stephenberry/glaze#1768] [Ideas]
Thanks for the great library.

It would be nice to be able to do this if it can be supported.
To support having an unknown_reader, unknown_writer and maybe others within the custom meta without having to redefine the entirety of the struct. 
```cpp
struct bigJson 
{
   /* ... */
};

template<>
struct glz::meta<bigJson>
{
    static constexpr auto unknown_read{ /* ... */ };
    static constexpr auto unknown_write{ /* ... */ };
};
```

This currently produces the following error on Visual Studio 2022 due to concept `detail::global_meta_t<T>` evaluating to false: 
`glaze\json\read.hpp(116,33): static_assert failed: 'unknown_read type not handled'`

**Comment by stephenberry** [comment]
Thanks for bringing this up. It would be nice to be able to specify only the unknown key map and reflect on all the other values automatically. I'll keep this issue alive, but it will probably be some time before it is addressed.

**Comment by stephenberry** [comment]
I'm moving this to discussions, because I think this issue requires more long term thought, especially as reflection in C++26 is coming. I don't want to overcomplicate the current reflection mechanisms that will need to be replaced.
